### PR TITLE
Deprecate Android BLE SDK

### PR DESCRIPTION
**Pull request by kstribrnAmzn**
*Description of changes:*
Add deprecation notice and file to library

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: DEPRECATED.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# ![image](https://user-images.githubusercontent.com/56273942/202568467-0ee721bb-1424-4efd-88fc-31b4f2a59dc6.png) DEPRECATED
+
+## Announcement:
+As of July 20th, 2023, this repository is deprecated. The contents of this repository will remain available but we will no longer provide updates or accept new contributions and pull requests. We recommend you start [here](https://docs.amplify.aws/start/q/integration/android/) for creating interactions between an Android device and AWS. For Bluetooth Low Energy APIs we recommend you start [here](https://developer.android.com/guide/topics/connectivity/bluetooth/ble-overview).
+
+Since the initial release of this package, several new Android version have release brining with them enhanced security and usability. AWS Amplify has also [release dozens of later versions](https://github.com/aws-amplify/aws-sdk-android/releases) of the [AWS SDK](https://github.com/aws-amplify/aws-sdk-android) with many new features and bug fixes.
```
Comment (cobusve) [inline DEPRECATED.txt:6]: several new Android versions have been released, bringing ...

Comment (cobusve) [inline DEPRECATED.txt:6]: AWS Amplify has also [released]

Comment (kstribrnAmzn) [reply DEPRECATED.txt:6]: Thanks for noticing this. I will fix this

Comment (kstribrnAmzn) [reply DEPRECATED.txt]: Fixing

**Review by AniruddhaKanhere — APPROVED** [review]

**Review by cobusve — APPROVED** [review]

**Pull request merged**